osf/migrations/0137_transfer_preprint_service_permissions.py

```diff
@@ -3,12 +3,9 @@
 from __future__ import unicode_literals
 
 from django.db import migrations
-from django.core.management.sql import emit_post_migrate_signal
```

Comment:
Why was this removed? Is it no longer supported / done automatically between migrations in Django2 / Py3?